olive/passes/pass_config.py

```diff
@@ -38,6 +40,7 @@ class PassConfigParam(ConfigParam):
     """
 
     searchable_values: SearchParameter = None
+    allowed_values: Set = None
```

Comment:
what's the field for?

Reply:
I think it is a user-facing change?

Reply:
This is used to validate param value. It's not a user facing change as this validation is internal only.

Reply:
PassConfigParam is used as user-facing stuff.

Reply:
I don't think so. `PassConfigParam` is defined by developer to indicate the parameter's attributes. As for users, they only need to provide a dict.